docs/testplan/platform/pmon-test-plan.md

```diff
@@ -74,49 +64,49 @@ CHASSIS_MODULE_INFO_SLOT_FIELD = 'slot'
 CHASSIS_MODULE_INFO_OPERSTATUS_FIELD = 'oper_status'
 ```
 
-### Test case # 4 – Data for ledd daemon
+### Test case # 3 – Data for ledd daemon
 #### Test objective
 Verify the expected data for ledd daemon (if not skip_ledd)
 #### Test steps
 * Verify the ledd running status
 
-### Test case # 5 – Data for pcied daemon
+### Test case # 4 – Data for pcied daemon
 #### Test objective
 Verify the expected data for pcied daemon (if not skip_pcied)
 #### Test steps
 * Verify the pcie config file "/usr/share/sonic/platform/pcie.yaml" exists
 * Verify the "PCIE_DEVICES" has "status" field set to "SUCCESS" 
```

Comment:
Please specify where this data resides, i.e., `Verify the "PCIE_DEVICES" has "status" field set to "SUCCESS" in State DB`